tests/conftest.py

```diff
@@ -198,7 +212,10 @@ def _setup_eng(num_subsystems, **kwargs):
 
 def pytest_runtest_setup(item):
     """Automatically skip tests if they are marked for only certain backends"""
-    allowed_backends = {"gaussian", "tf", "fock"}
+    if tf_available:
```

Comment:
it seems to have been a lucky accident that this fixture was already here :dango: